docs/lists_degrees.md

```diff
@@ -199,6 +199,60 @@ This list is [autocomplete compatible](autocomplete_compatability.md).
 | `hecos_code` | optional string | The HESA [DEGSBJ](https://www.hesa.ac.uk/collection/c22053/e/degsbj) code for this degree subject (from [HECoS](https://www.hesa.ac.uk/support/documentation/hecos)) (for single subjects) |
 | `subject_ids` | optional UUID array | The `SINGLE_SUBJECTS` IDs of the individual parts, in order (for combined subjects) |
 
+### `DfE::ReferenceData::Degrees::GENERIC_SUBJECTS`
+
+```ruby
+require 'dfe/reference_data/degrees'
+```
+
+Generic subject options
+
+Owner: Register team.
+
+Users: Register team.
+
+Source: Manual
+
+Quality: Manually updated on an ad-hoc basis. Please submit a pull request if inaccuracies or omissions are found.
+
+This list is [autocomplete compatible](autocomplete_compatability.md).
+
+| Field | Type | Purpose |
+|---|---|---|
+| `id` | UUID | A unique identifier. The same as `dttp_id` if that field is non-`nil`, otherwise a new UUID was minted at import time. |
+| `name` | string | The long name of the subject, eg "accountancy" |
+| `suggestion_synonyms` | string array | A list of common alternative names that *may* be appropriate for this subject |
+| `match_synonyms` | string array | A list of common alternative names that are equivalent to this subject. An answer matching a match synonym can be safely matched to this subject.|
```

Comment:
Drop these for the generic subjects? 🤔 

```suggestion
```

Reply:
I've got some synonyms to add for them later. eg `n/a`, `na`, `other`...

Reply:
how would you use these? Do these options appear in an autocomplete for you?